controller/src/beerocks/cli/beerocks_cli_bml.h

```diff
@@ -205,6 +205,8 @@ class cli_bml : public cli {
                               const std::string &channel_pool);
     int get_dcs_scan_results(const std::string &radio_mac, uint32_t max_results_size,
                              bool is_single_scan = false);
+    template <typename T>
+    const std::string string_from_int_array(T *arr, size_t arr_max_size);
```

Comment:
Why does it need to be exported now? in other words, why is it extern instead of static and why is it declared in the header file? It is not explained in the commit message.